pyart/graph/gridmapdisplay.py

```diff
@@ -1110,6 +1112,46 @@ def cartopy_coastlines(self):
             category="physical", name="coastline", scale="10m", facecolor="none"
         )
 
+    def plot_maxcappi(
```

Comment:
More curiosity, but reason for having the code in its own file and then here as well?

Reply:
It was a long code, didn't wanted to overcrowd the gridmapdisplay.py script

Reply:
What do you suggest?

Reply:
I think it would be fine being in the gridmapdisplay, or either or, we can get @mgrover1 thoughts on it as well.

Reply:
I’m open to merging it back if that’s preferred. Let's see what @mgrover1 has to say!